<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraWindow</class>
 <widget class="QDialog" name="CameraWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FerromagnetWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background: rgba(214, 219, 233,255);
}

QComboBox{
background: rgba(214, 219, 233,255);
border: none;
}

QComboBox:hover{
background: rgba(255, 255, 191,255);
border:  2px solid rgba(255, 170, 0,255);
}

QPushButton:hover{
background: rgba(255, 255, 191,255);
border:  2px solid rgba(255, 170, 0,255);
}

QPushButton{
border: none;
background: rgba(214, 219, 233,255);
}

QTableWidget {
    background: rgba(214, 219, 233,255);
border: none;
}

QHeaderView::section{
background-color: rgba(235, 241, 255, 255);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,2,1,2,2,2,2,1">
     <item>
      <widget class="QComboBox" name="comboBoxViewType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelViewOffset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>center of view</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomTableWidget" name="tableWidgetCamOffset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCameraAngle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>relative placement of camera (spherical coordinates)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomTableWidget" name="tableWidgetPos">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBase">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>show base plane</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomTableWidget" name="tableWidgetBaseOffset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomTableWidget" name="tableWidgetBaseColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
       <item>
        <widget class="QPushButton" name="pushButtonApply">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOk">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomTableWidget</class>
   <extends>QTableWidget</extends>
   <header>CustomTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>on_push_button_apply()</slot>
 </slots>
</ui>
